packages/xo-web/src/xo-app/jobs/schedules/index.js

```diff
@@ -204,8 +209,15 @@ export default class Schedules extends Component {
     },
   ]
 
+  _getTimezone = createSelector(
+    () => this.state.schedule !== undefined,
+    () => this.state.timezone,
+    (scheduleExists, timezone) => (scheduleExists ? timezone : DEFAULT_TIMEZONE)
+  )
```

Comment:
This seems a bit weird.

What's the wanted behavior?

Reply:
If schedule is missing means that the user want to create the schedule for the first time so we pas the browsers time zone as default value. If schedule exists and the timezone is undefined, the timezone default value will be the servers timezone. I did the same implementation that the  `backup/new`.

Reply:
Simply set the initial state at the default value.